test/unit/migrations/048-test.js

```diff
@@ -60,14 +76,40 @@ describe('migration #48', function () {
 
     const newStorage = await migration48.migrate(oldStorage)
     assert.deepEqual(newStorage.data, {
+      ...expectedPreferencesState,
+      NetworkController: {
+        foo: 'bar',
+      },
+      foo: 'bar',
+    })
+  })
+
+  it('should delete NetworkController.provider if the type is "localhost"', async function () {
```

Comment:
I guess the alternative would be to migrate localhost types, but my guess is we don't have a chainId recorded for these so we can't easily? Could we treat them the same way we do other RPCs with no chainId?

Reply:
AFAIK, the only network with a `localhost` provider type is the `Localhost 8545` network we hard-coded into the extension.

Reply:
So, since we know everything about that network, we're effectively migrating the sole instance of `localhost` networks.